src/gitCommands.ts

```diff
@@ -91,3 +92,31 @@ export const findModifiedFiles = (): string[] | null => {
 		return null;
 	}
 };
+
+export const findLastlyModifiedFile = async (): Promise<string | null> => {
+	try {
+		const modifiedFiles = execSync('git ls-files --modified', {
+			encoding: 'utf-8',
+		})
+			.trim()
+			.split('\n');
+		if (modifiedFiles.length === 0) {
+			return null;
+		}
+		let lastlyModifiedFile = modifiedFiles[0];
```

Comment:
IMO better to make it `string | null` and by default it is `null`

Reply:
addressed